MAUI/DataForm/grouping.md

```diff
@@ -0,0 +1,308 @@
+
+{% tabs %}
+{% highlight c# %}
+public class ContactInfo
+{
+    [Display(GroupName = "Name", Order = 0)]
+    public string FirstName { get; set; }
+
+    [Display(GroupName = "Name", Order = 1)]
+    public string LastName { get; set; }
+
+    [Display(GroupName = "Name", Order = 2)]
+    public string MiddleName { get; set; }
+}
+{% endhighlight %}
+{% endtabs %}
+
+The order of the `DataFormGroupItem` can also be changed by handling the `GenerateDataFormItem` event.
+
+{% tabs %}
+{% highlight c# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+        private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.GroupName == "Name")
+            {
+                if (e.DataFormItem.FieldName == "FirstName")
+                {
+                    e.DataFormItem.RowOrder = 0;
+                }
+                else if (e.DataFormItem.FieldName == "LastName")
+                {
+                    e.DataFormItem.RowOrder = 2;
+                }
+                else if (e.DataFormItem.FieldName == "MiddleName")
+                {
+                    e.DataFormItem.RowOrder = 1;
+                }
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+
+## Changing group name
+
+The `GroupName` for the group can be changed in the `GenerateDataFormItem` event.
+
+{% tabs %}
+{% highlight c# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+       private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem != null)
+            {
+                if (e.DataFormItem.GroupName == "Name")
+                {
+                    e.DataFormItem.GroupName = "Name Group";
+                }
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Changing the layout for the group
+
+The linear or grid layout for the particular group can be loaded by handling the `GenerateDataFormItem` event.
+By setting the `ColumnCount` property in the data form, non-grouped items only will be arranged in the grid layout. To load the grid layout, set the [ColumnCount] for the `DataFormGroupItem`
+
+{% tabs %}
+{% highlight c# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormGroupItem != null)
+        {
+            if (e.DataFormGroupItem.Name == "Name")
+            {
+                e.DataFormGroupItem.ColumnCount = 2;
+            }
+            else if (e.DataFormGroupItem.Name == "Details")
+            {
+                e.DataFormGroupItem.ColumnCount = 3;
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Changing linear and grid layout for the group
```

Comment:
## Changing linear and grid layout for the group

Remove this as explained in the above topic

Reply:
removed